src/www/services_dhcpv6.php

```diff
@@ -60,11 +156,28 @@ function reconfigure_dhcpd()
     }
     if (!empty($_POST['act'])) {
         $act = $_POST['act'];
-    } else {
-        $act = null;
     }
 }
 
+/**
+ * XXX: In case of tracking, show different form and only handle on/off options.
+ *      this code injection is intended to keep changes as minimal as possible and avoid regressions on existing isc-dhcp6 installs,
+ *      while showing current state for tracking interfaces.
+ **/
+if ($if === null) {
+    /* if no interface is provided this invoke is invalid */
+    header(url_safe('Location: /index.php'));
+    exit;
+} elseif (!empty($config['interfaces'][$if]['track6-interface']) && !isset($config['interfaces'][$if]['dhcpd6track6allowoverride'])) {
+    if ($_SERVER['REQUEST_METHOD'] === 'GET') {
+        show_track6_form($if);
+    } elseif ($_SERVER['REQUEST_METHOD'] === 'POST') {
+        process_track6_form($if);
+    }
+    die;
```

Comment:
```suggestion
    exit;
```

I would reserve "die" for actual errors or aborts